pkgs/development/python-modules/uvicorn/default.nix

```diff
@@ -29,11 +33,18 @@ buildPythonPackage rec {
     wsproto
   ];
 
-  checkPhase = ''
-    $out/bin/uvicorn --help
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace "h11==0.8.*" "h11"
   '';
 
-  patches = [ ./setup.patch ];
```

Comment:
You should also remove the `setup.patch` file.

Reply:
done